dts/arm64/intel/intel_socfpga_agilex5.dtsi

```diff
@@ -239,4 +239,39 @@
 		status = "disabled";
 	};
 
+	dma0: dma@10DB0000 {
+		compatible = "snps,designware-dma-axi";
+		#dma-cells = <1>;
+		reg = <0x10DB0000 0x1000>;
+		interrupt-parent = <&gic>;
+		interrupts = <GIC_SPI 81 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+			     <GIC_SPI 82 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+			     <GIC_SPI 83 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>,
+			     <GIC_SPI 84 IRQ_TYPE_LEVEL
+				 IRQ_DEFAULT_PRIORITY>;
+		dma-channels = <4>;
+		resets = <&reset RSTMGR_DMA_RSTLINE>;
+		status = "disabled";
+	};
+
+	dma1: dma@10DC0000 {
+		compatible = "snps,designware-dma-axi";
+		#dma-cells = <1>;
+		reg = <0x10DC0000 0x1000>;
+		interrupt-parent = <&gic>;
+		interrupts = <GIC_SPI 166 IRQ_TYPE_LEVEL
```

Comment:
we can add these in one single line if they are with in the limit - 100 lines column